addons/account/wizard/accrued_orders.py

```diff
@@ -149,9 +148,8 @@ def _ellipsis(string, size):
                 order_line = order.order_line[0]
                 account = self._get_computed_account(order, order_line.product_id, is_purchase)
                 distribution = order_line.analytic_distribution if order_line.analytic_distribution else {}
-                if not is_purchase and order.analytic_account_id:
-                    analytic_account_id = str(order.analytic_account_id.id)
-                    distribution[analytic_account_id] = distribution.get(analytic_account_id, 0) + 100.0
+                if not is_purchase:
+                    self._update_distribution(distribution, order)
```

Comment:
I'm not sure I understand what you're doing here.
The idea before this PR was to add the account of the SO(s) to the distribution of the accrued entries. Pretty much the same way we added the account of the SO to the distribution of the SOLs when invoicing (except you can mix several orders).

But now, you add directly the distribution of the project on the SOL. Why are you again adding it now for accruals? 
You don't when invoicing, but you do when doing accruals.
I feel like you don't need to do all this, it's already done on the SOL level.
You could probably remove all the `_update_distribution`.

Reply:
We made this adaption because we replaced the 'analytic_account_id' of the SO by a 'project_id' field. So we thought we had to adapt this part of the code as it seems to use the 'analytic_account_id' of the SO which does not exist anymore.. What should we do then?

Reply:
Well, if I understand correctly, in most cases, a SO that has a project_id should already have it's account(s) into the distribution of the SOL. Before, that wasn't the case.
So we don't need to re-add it again.
I would just remove all the `_update_distribution`.

Reply:
Of course, sometimes, the distribution of the project is not given (because it was already computed by the distribution models), but I would say it does not matter. It's better to not include it than to include it twice

Reply:
Alright, it makes sense :) We will do the changes, thank you for your help!